### ISSUE TITLE
Finalize Food Bank contract

### ISSUE THREAD

**Issue by lippytak** [codeforamerica/clean#242]
@RebeccaCoelius assuming we don't need to make any changes just feel free to send the signed docs directly to Liliana and we'll consider this done!

**Comment by RebeccaCoelius** [comment]
Meeting w/ Meghan 1/21

**Comment by RebeccaCoelius** [comment]
Meghan completing paperwork today/tomorrow

**Comment by RebeccaCoelius** [comment]
Rebecca + Jake visiting food bank in person 1/27

**Comment by lippytak** [comment]
Waiting on FB to decide if/how they want to do this. I'll ping them in a couple days and open another issue if we need to do something.
